docs/documentation/development/bindings/profiles.md

```diff
@@ -89,4 +89,29 @@ Two methods have to be implemented to achieve this:
 
 ```java
 <item-type> MyItem { channel="<bindingID>:<thing-typeID>:MyThing:myChannel"[profile="MyScope:MyProfile"]}
-```
\ No newline at end of file
+```
+
+## Existing Profiles
+
+### OffsetProfile
+
+The `OffsetProfile` provides the possibility to adjust a value from a device before it arrives at the framework.
+An offset can be specified via the parameter `offset` which has to be a `QuantityType` or `DecimalType`.
+A positive offset is the amount of change from the device towards the framework and vice versa.
```

Comment:
No, not "vice versa". You'll have to stick to explaining a single direction if this sentence should be of any help.